e2store/src/era.rs

```diff
@@ -50,7 +50,7 @@ impl Era {
 
         // Iterate over the block entries. Skip the first and last 3 entries.
         let mut next_slot = slot_index_block.slot_index.starting_slot;
-        for idx in 1..entries_length - 4 {
+        for idx in 1..entries_length - 3 {
             let entry: Entry = file.entries[idx].clone();
```

Comment:
nit: While you are at it, this cloning seems completely unnecessary (we can just pass `&file.entries[idx]` below).